tests/Auth0Test.php

```diff
@@ -554,27 +545,6 @@ public function testThatIdTokenIsPersistedWhenSet()
         $this->assertEquals($id_token, $_SESSION['auth0__id_token']);
     }
 
-    /**
-     * @throws CoreException
-     */
-    public function testThatEmptyApplicationNonceFailsIdTokenValidation()
```

Comment:
shouldn't this one be kept but with the condition/check inverted? e.g. id token validation when no nonce is stored should pass

Reply:
That's handled in `testThatRenewTokensSucceeds`